v3/_locales/en/messages.json

```diff
@@ -98,11 +98,11 @@
   "testUATitle": {
     "message": "Test your user-agent string"
   },
-  "considerContainers": {
```

Comment:
This removal is irrelevant to the merge request.

Reply:
Formatting was done by vscode. Also some changes may have been mistake from my side sorry for that